imports/plugins/core/product/server/no-meteor/schemas/schema.graphql

```diff
@@ -6,3 +6,68 @@ type ProductConfiguration {
   "The ProductVariant ID"
   productVariantId: ID!
 }
+
+"Operation"
+type Operation {
+  "Id of document"
+  _id: String
+}
+
+"A bulk write error type"
+type WriteError {
+  "Error code"
+  code: Int
```

Comment:
These are straight from Mongo? I think we should keep such things internal.

Reply:
Yes, kept internal now.